src/assets/misc/connectingChains/connectingChains.asset.ts

```diff
@@ -0,0 +1,107 @@
+import { ItemInteractionType } from 'pandora-common';
+
+DefineAsset({
+	name: 'Connecting Chains',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		Chains: {
+			name: 'Chains',
+			default: '#DDDDDD',
+		},
+	},
+	// size:400, y:550, centered
+	preview: 'preview.png',
+	modules: {
+		lock: {
+			type: 'lockSlot',
+			name: 'Lock',
+			occupiedProperties: {
+				blockAddRemove: true,
+			},
+		},
+		chains: {
+			type: 'typed',
+			name: 'Connecting Chains',
+			interactionType: ItemInteractionType.ADD_REMOVE,
+			variants: [
+				{
+					id: 'none',
+					name: 'No Chain',
+					default: true,
+				},
+				{
+					id: 'beltChain',
+					name: 'Belt Chains',
+					switchMessage: 'SOURCE_CHARACTER connects the cuffs to TARGET_CHARACTER_DYNAMIC_POSSESSIVE belt with short chains.',
+					properties: {
+						poseLimits: {
+							arms: {
+								position: 'back',
+							},
+							bones: {
+								arm_l: 70,
+								elbow_l: 20,
+								arm_r: 70,
+								elbow_r: 20,
+							},
+						},
+						attributes: {
+							provides: [
+								'Restraint',
+								'Restraint_arms',
+							],
+							requires: [
+								'Wrist_cuffs_chainable',
+								'Belt_chainable',
+							],
+						},
+					},
+				},
+				{
+					id: 'armbinderChain',
+					name: 'Kneeling Chains',
+					switchMessage: 'SOURCE_CHARACTER connects the cuffs to TARGET_CHARACTER_DYNAMIC_POSSESSIVE armbinder with short chains.',
+					properties: {
+						poseLimits: {
```

Comment:
Please test the front view, too.
In the front view one can still see the chains and it looks clearly wrong.

The best way is to hide the chain images when the view is front and add new chain images for just the front view.
I do not think that you need many new chain variants to show only in the front facing view. Maybe one or two should be enough so that they disappear behind the legs, but do not clip through on the other side when manually dragging the slider from 5 to -27.

Reply:
Good find. I really haven't thought about this.